packages/config/src/templates/opStack.ts

```diff
@@ -905,6 +941,37 @@ The Kailua state validation system is primarily optimistically resolved, so no v
         ],
       }
     }
+    case 'OpSuccinctFDP': {
+      return {
+        categories: [
+          {
+            title: 'Validity proofs',
+            description:
+              'State roots are proposed by whitelisted proposers who create dispute games via the DisputeGameFactory by posting a bond. Once created, the game enters a challenge period during which whitelisted challengers can dispute the proposal by also posting a bond. If challenged, anyone can submit a ZK proof to prove the correct state within the proving period. After the challenge period passes without a successful challenge, or after a valid proof is submitted, anyone can resolve the game and finalize the state root.',
```

Comment:
I would make this explanation more precise by pulling challenge period and proving period from the contract (`maxChallengeDuration` and `maxProveDuration` from `OPSuccinctFaultDisputeGame`). Idk if we need extra steps to make it readable (3.5 days and 1 day).